apps/site/test/site/trip_plan/merge_test.exs

```diff
@@ -16,8 +16,8 @@ defmodule Site.TripPlan.MergeTest do
 
   describe "merge/3" do
     test "includes no duplicates" do
-      ptest accessible: list(of: positive_int(), max: 5),
-            unknown: list(of: positive_int(), max: 5) do
+      ptest accessible: list(of: positive_int(), min: 1, max: 5),
+            unknown: list(of: positive_int(), min: 1, max: 5) do
```

Comment:
Does this mean it fails if one of the two lists is empty? Is that a problem?

Reply:
Good question -- yes It was failing when `unknown` was an empty list, and in trying to assess whether that's a problem or not, at the very least for the intent of this test it doesn't seem to be an issue. 

But I am reverting as it's not decisively an improvement :shrug: